hippo-digital/svp-tech-poc
# Task/returning user able to toggle on off food parcels

**Note: This should be merged AFTER #3**

## Summary of changes

It is now possible to toggle on/off food parcels.

## Testing instructions

Run the following commands:

```bash
docker pull amazon/dynamodb-local
docker run -d -p 8000:8000 amazon/dynamodb-local
```

Then fire up the app as usual, log in using an NHS account, and fill out the presented form. After hitting 'Submit' on the check your answers page the record will be pushed to the database and you will be routed back to the index page.

If you then log in again with the same account, rather than being shown the form, you will be redirected to a 'show saved answers page', which displays the summary list.

Next to the "Help with food delivery required" icon there is a 'change' link. Clicking that link, changing the supplied answer, clicking 'submit and next' then on the show saved answers page clicking 'save  changes' will persist any changes made to the backend. You can verify they have been persisted by logging out and logging in again.
